ci-operator/config/openshift/origin-aggregated-logging/openshift-origin-aggregated-logging-master.yaml

```diff
@@ -199,6 +201,8 @@ tests:
         name: pipeline:logging-curator5
       - env: IMAGE_LOGGING_FLUENTD
         name: pipeline:logging-fluentd
+      - env: IMAGE_ELASTICSEARCH_OPERATOR
```

Comment:
do you also need to add `IMAGE_ELASTICSEARCH_OPERATOR_REGISTRY` ? just like for CLO?

Reply:
I believe its fine based on what I see in the test log which references an image from the pipeline:
![image](https://user-images.githubusercontent.com/4548408/111516327-268bbb80-872a-11eb-90fb-bf339acc6230.png)